addons/web/static/tests/views/kanban_tests.js

```diff
@@ -193,6 +193,82 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('basic grouped rendering with active field and archive enabled (archivable true)', function (assert) {
+        assert.expect(9);
+
+        // add active field on partner model and make all records active
+        this.data.partner.fields.active = {string: 'Active', type: 'char', default: true};
+
+        var envIDs = [1, 2, 3, 4]; // the ids that should be in the environment during this test
+        var kanban = createView({
+            View: KanbanView,
+            model: 'partner',
+            data: this.data,
+            arch: '<kanban class="o_kanban_test" archivable="true">' +
+                        '<field name="active"/>' +
+                        '<field name="bar"/>' +
+                        '<templates><t t-name="kanban-box">' +
+                        '<div><field name="foo"/></div>' +
+                    '</t></templates></kanban>',
+            groupBy: ['bar'],
+            intercepts: {
```

Comment:
hmm also, you can remove this part I think.  This is not really necessary for the test, now that I think of it